include/PropertyDescription.h

```diff
@@ -13,6 +13,36 @@
 
 namespace ChimeraTK {
 
+  /// configuration choices for writing persistance files for DOOCS properties
+  struct PersistConfig {
+    enum {
+      OFF = 0,
+      ON = 1,  // write long arrays to separate files below hist/, short arrays to config file
+      AUTO = 2 // default DOOCS behaviour: persist only short arrays, in config file
+    };
+    int val;
+
+    PersistConfig(std::string txt) {
+      transform(txt.begin(), txt.end(), txt.begin(), ::tolower);
+      if(txt == "false" or txt == "0") {
```

Comment:
maybe allow `"on"` and `"off"` as well, since those are the values of the enum?

Reply:
fixed